packages/kcms/src/team/adaptor/validator/match.ts

```diff
@@ -0,0 +1,38 @@
+import { z } from '@hono/zod-openapi';
+
+const finishStatus = ['goal', 'retired'] as const;
+
+const teamSchema = z.object({
+  id: z.string().openapi({ example: '45098607' }),
+  teamName: z.string().openapi({ example: 'チーム1' }),
+});
+
+const runResultSchema = z.object({
+  id: z.string().openapi({ example: '60980640' }),
+  teamId: z.string().openapi({ example: '45098607' }),
+  points: z.number().openapi({ example: 4 }),
+  finishedTimeSeconds: z.number().openapi({ example: 30 }),
```

Comment:
```suggestion
  goalTimeSeconds: z.number().nullable().openapi({ example: 30 }),
```
念のため; **ゴール**と**フィニッシュ**は別物で、**フィニッシュ**ではタイムは記録されません(nullになります)。